### PR TITLE
Updated the faq page

### PR DESCRIPTION
**Pull request by qstokkink**
Fixes https://github.com/Tribler/tribler/issues/5604

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -119,19 +119,19 @@ <h1>Frequently Asked Questions - FAQ</h1>
 				<h2>Downloading and Uploading</h2>
 				<div class="faq_item">
 					<div class="faq_question">How do I download using Tribler?</div >
-					<div class="faq_answer"><p>The easiest way to download using Tribler is to search from the search panel at the top of the screen. Just type in what you're looking for and press the ENTER button. You will be taken to the Results tab and your search results will start coming in. When you've found what you're looking for you can click the Download button directly or first click on the search result to examine it in detail and then click the Download button.</p>
+					<div class="faq_answer"><p>The easiest way to download using Tribler is to search from the search panel at the top of the screen. Just type in what you're looking for and press the ENTER button. You will be taken to the Results tab and your search results will start coming in. When you've found what you're looking for you can double-click search result to show the download details. If you're happy with the anonymization settings (ON by default), you can click the button to start downloading.</p>
```
Comment (egbertbouman) [inline faq.html:122]: Search results no longer need double clicking.

File: faq.html [hunk 2]
```diff
@@ -119,19 +119,19 @@ <h1>Frequently Asked Questions - FAQ</h1>
...
 					<p>For a step by step introduction, see the example below:<ol>
-					<li>Type Sintel in the search bar and click on the Search button; (Visit the Sintel website for more information about the movie.)</li>
-					<li>The results view should come forward and give you a number of results. One of them should be named Sintel. You can now click the download button to the right.</li>
+					<li>Type Sintel in the search bar and press ENTER; (Visit the Sintel website for more information about the movie.)</li>
+					<li>The results view should come forward and give you a number of results. At least one of them should be named Sintel. You can now double-click, which will show you the download information dialog.</li>
```
Comment (egbertbouman) [inline faq.html:125]: Same here.

**Review by egbertbouman — COMMENTED** [review]
Looks good, except for the 2 minor comments.

**Comment by qstokkink** [comment]
@egbertbouman done 👍

**Review by egbertbouman — APPROVED** [review]

**Pull request merged**